x-pack/test/functional/services/ml/common_data_grid.ts

```diff
@@ -58,41 +60,46 @@ export function MachineLearningCommonDataGridProvider({ getService }: FtrProvide
       return rows;
     },
 
-    async getDataGridRows(tableSubj: string) {
+    async getDataGridCells(tableSubj: string) {
       return (await testSubjects.find(tableSubj)).findAllByTestSubject('dataGridRowCell');
     },
 
     async assertEuiDataGridNotEmpty(tableSubj: string) {
-      const dataGridRows = await this.getDataGridRows(tableSubj);
+      const dataGridRows = await this.getDataGridCells(tableSubj);
       expect(dataGridRows.length).to.be.greaterThan(
         0,
         `EuiDataGrid '${tableSubj}' should have at least one row (got '${dataGridRows.length}')`
       );
     },
 
     async assertEuiDataGridEmpty(tableSubj: string) {
-      const dataGridRows = await this.getDataGridRows(tableSubj);
+      const dataGridRows = await this.getDataGridCells(tableSubj);
       expect(dataGridRows.length).to.eql(
         0,
         `EuiDataGrid '${tableSubj}' should be empty (got '${dataGridRows.length} rows')`
       );
     },
```

Comment:
Nit: For consistency, I'd suggest to also rename local variables from `dataGridRows` to `dataGridCells` and to adjust the assertion messages to mention cell/cells instead of row/rows.

Reply:
Renamed here https://github.com/elastic/kibana/pull/125162/commits/d4954ef46b88473740d8b9ab25e46bccf8425cef